conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -38,18 +41,17 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-
+conda install --yes conda-build==1.18.2
```

Comment:
Could you help @frol and I understand this change? What does/did it solve? Is it still needed?